src/main.rs

```diff
@@ -41,11 +69,88 @@ fn main() -> ! {
     // or remove it and set ESP_LOGLEVEL manually before running cargo run
     // this requires a clean rebuild because of https://github.com/rust-lang/cargo/issues/10358
     esp_println::logger::init_logger_from_env();
-    info!("Logger is setup");
+    log::info!("Logger is setup");
     {% endif -%}
-
     println!("Hello world!");
+    {% if wifi -%}
+    {% if arch == "riscv" -%}
+    let timer = SystemTimer::new(peripherals.SYSTIMER).alarm0;
+    {% else -%}
+    let timer = TimerGroup::new(
+        peripherals.TIMG1,
+        &clocks,
+        &mut system.peripheral_clock_control,
+    )
+    .timer0;
+    {% endif -%}
+    let init = initialize(
+        EspWifiInitFor::Wifi,
+        timer,
+        Rng::new(peripherals.RNG),
+        system.radio_clock_control,
+        &clocks,
+    )
+    .unwrap();
```

Comment:
Haven't thought about it before but I like the idea

Reply:
It would make things simpler, which is a good thing! Shall I simplify the code this way? We can use a [rhai post script](https://cargo-generate.github.io/cargo-generate/templates/scripting.hook-types.html#post) to print some links to examples if the user chose to use `esp-wifi`